### ISSUE TITLE
ncu -n indicates webpack but not webpack-dev-server

### ISSUE THREAD

**Issue by jbruni** [tjunnone/npm-check-updates#286]
I have this in my `package.json`:

```
    "webpack": "^1.13.2",
    "webpack-dev-server": "^1.16.2",
```

When I run `ncu -n`, it brings:

```
 webpack  ^1.13.2  →  ^2.1.0-beta.25 
```

But it does not bring webpack-dev-server's 2.1.0-beta.8

References:
- [webpack's package.json](https://github.com/webpack/webpack/blob/master/package.json)
- [webpack-dev-server's package.json](https://github.com/webpack/webpack-dev-server/blob/master/package.json)

**Comment by raineorshine** [comment]
I get the following:

```
 webpack             ^1.13.2  →  ^2.1.0-beta.25 
 webpack-dev-server  ^1.16.2  →   ^2.1.0-beta.9 
```

It may not be showing the webpack-dev-server upgrade if you already have it installed. What do you get when you run `npm ls --depth=0`?

**Comment by raineorshine** [comment]
If that's the case, it may be related to this known bug: #263

**Comment by jbruni** [comment]
Thanks for you reply.

I had seen the referenced bug (#263) before I opened this issue. I have deleted node_modules folder - `rm -rf node_modules` - and tried again, and it didn't change anything at that time.

I just tried again and it gave same result as yours! And it is fun that I did it **before** coming back to GitHub, and reading your feedback. I was going to report the news!

I guess it **was** a "dependency-bug" and that in the meantime **they** resolved it when releasing the fresh new "beta.9" version. (It was "beta.8" this morning.)

Maybe they read this issue report and fixed on their side. :-)

Anyway, it was a pleasure having this talk with you. I use `ncu` daily, as I want to be using latest releases. I already recommended it to a number of people.

Thank you.
